README.md

```diff
@@ -1,6 +1,8 @@
 coala-vim: coala extension for vim/gvim
 =======================================
 
+### Note: We are moving away from this plugin. [neomake](https://github.com/neomake/neomake) is a better alternative with more features.
```

Comment:
Actually, you just need to add one line to your vimrc when you have neomake installed to make coala work. So actually not much work needed. I'll figure out what that line is a little later :)